### PR TITLE
Newsweek Clone

### PR DESCRIPTION
**Pull request by hamdiheb**

### CODE REVIEW

**Review by mricanho — CHANGES_REQUESTED** [review]
# 🔴  Changes Required 🔴  

Hello @hamdiheb @fortuneonyeka, you did a good job in this milestone🔥, just a few changes are required:

- [x]  **Your nav is not so good looking right now, remember that you are cloning the NW website, so you need to put extra attention to the details**
- [x] **You need to add a logo in the nav, there are plenty NW logos on the internet that you can use** 
- [x] **Double check the size of the text in the nav, also, align the items**
- [x] **You missed the "Login" in the nav, please add it**
- [x] **The hamburger button only needs to be in the mobile version**
- [x] **The search input and the search icon are not aligned, please fix that**
- [x] **You have an underline in all the links, remove it, and check the transition color for every case**
- [x] **Check the margin and the color of the social icons in the footer**

As described in the Code reviews limit [Policy ](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have 2 code reviews left for this project.

work hard

You can contact me on slack if you have any questions just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!👨🏽‍💻  🖥️

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  .search {
+    width: 80px;
+    outline: none;
+    border: none;
+  }
+
+  .search_icon {
+    display: flex;
+    float: right;
+    position: relative;
```
Comment (smcommits) [inline css/index.css:146]: You should use bootstrap classes for these.

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    height: 10px;
+    bottom: 18px;
+    left: 5px;
+  }
+
+  .nav-fast-link {
+    display: flex;
```
Comment (smcommits) [inline css/index.css:153]: You should use bootstrap classes for this as well.

**Review by smcommits — APPROVED** [review]
# Status: Approved :+1: 

## Hello, @hamdiheb and @fortuneonyeka! You guys have done a great job. The components look great. Nice work implementing the changes that the previous TSE required. This PR is now approved. You can go ahead and merge the branch. 

## Optional

## Please note that the following suggestion is optional but highly recommended. 

- [ ] **You should use more bootstrap classes. Please refer to the inline comments**

There are plenty of declarations in your CSS that can be easily achieved with Bootstrap. Not using it defeats the purpose of using a framework. I have highlighted some of the rules in the inline comments.

File: css/index.css (hunk 2 above)
Comment (hamdiheb) [reply css/index.css:153]: Thank you so much for approving our project , and thank you so much for this advice it's really helpful

**Pull request merged**